flake.nix

```diff
@@ -134,16 +144,9 @@
             ];
           };
 
-          homestakeros = pkgs.mkYarnPackage {
-            pname = "homestakeros";
-            version = "0.0.1";
-
-            src = ./.;
-            packageJSON = ./package.json;
-            yarnLock = ./yarn.lock;
-            yarnNix = ./yarn.nix;
-          };
-          default = packages.homestakeros;
+          frontend = inputs'.frontend.packages.default;
+          backend = inputs'.backend.packages.default;
```

Comment:
packages.${pkgs.system}.default

Reply:
Does not need to be clarified when using `inputs'` or `self'`. Ref: https://flake.parts/module-arguments#inputs